doc/overview.rst

```diff
@@ -162,112 +261,24 @@ Here is an example of how to define metrics:
-
-
-If you have an existing model and want to deploy it locally, don't specify a sagemaker_session argument to the ``MXNetModel`` constructor.
-The correct session is generated when you call ``model.deploy()``.
-
-Here is an end-to-end example:
-
-.. code:: python
-
-    import numpy
-    from sagemaker.mxnet import MXNetModel
-
-    model_location = 's3://mybucket/my_model.tar.gz'
-    code_location = 's3://mybucket/sourcedir.tar.gz'
-    s3_model = MXNetModel(model_data=model_location, role='SageMakerRole',
-                          entry_point='mnist.py', source_dir=code_location)
-
-    predictor = s3_model.deploy(initial_instance_count=1, instance_type='local')
-    data = numpy.zeros(shape=(1, 1, 28, 28))
-    predictor.predict(data)
-
-    # Tear down the endpoint container and delete the corresponding endpoint configuration
-    predictor.delete_endpoint()
-
-    # Deletes the model
-    predictor.delete_model()
-
-
-If you don't want to deploy your model locally, you can also choose to perform a Local Batch Transform Job. This is
-useful if you want to test your container before creating a Sagemaker Batch Transform Job. Note that the performance
-will not match Batch Transform Jobs hosted on SageMaker but it is still a useful tool to ensure you have everything
-right or if you are not dealing with huge amounts of data.
-
-Here is an end-to-end example:
-
-.. code:: python
-
-    from sagemaker.mxnet import MXNet
-
-    mxnet_estimator = MXNet('train.py',
-                            role='SageMakerRole',
-                            train_instance_type='local',
-                            train_instance_count=1,
-                            framework_version='1.2.1')
-
-    mxnet_estimator.fit('file:///tmp/my_training_data')
-    transformer = mxnet_estimator.transformer(1, 'local', assemble_with='Line', max_payload=1)
-    transformer.transform('s3://my/transform/data, content_type='text/csv', split_type='Line')
-    transformer.wait()
+BYO Docker Containers with SageMaker Estimators
+-----------------------------------------------
 
-    # Deletes the SageMaker model
-    transformer.delete_model()
+To use a Docker image that you created and use the SageMaker SDK for training, the easiest way is to use the dedicated ``Estimator`` class.
+You can create an instance of the ``Estimator`` class with desired Docker image and use it as described in previous sections.
 
+Please refer to the full example in the examples repo:
 
-For detailed examples of running Docker in local mode, see:
+::
 
-- `TensorFlow local mode example notebook <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/tensorflow_distributed_mnist/tensorflow_local_mode_mnist.ipynb>`__.
-- `MXNet local mode example notebook <https://github.com/awslabs/amazon-sagemaker-examples/blob/master/sagemaker-python-sdk/mxnet_gluon_mnist/mnist_with_gluon_local_mode.ipynb>`__.
+    git clone https://github.com/awslabs/amazon-sagemaker-examples.git
```

Comment:
I wonder if it might be better to rephrase these to instruct how to find them in a SageMaker Notebook Instance instead

Reply:
How about adding that? I think that there is a category of customers that wants to be able to look at a notebook without the overhead of starting a notebook instance.

Reply:
fair enough